test/oracle.test.js

```diff
@@ -158,16 +158,21 @@ describe('lazyConnect', function() {
       lazyConnect: false,
       debug: false,
     };
-    var ds = getDS(dsConfig);
 
-    ds.on('error', function(err) {
-      err.message.should.containEql('ECONNREFUSED');
+    try {
+      getDS(dsConfig);
+      // Make sure it throws err
+      true.should.not.be.ok;
```

Comment:
@pthieu I just realized that I checked the wrong doc page every time I use `should` :(